scripts/core/editor3/actions/editor3.tsx

```diff
@@ -236,8 +239,10 @@ export function changeCase(changeTo: ITextCase, selection: SelectionState) {
 
 export type EditorLimit = { ui: CharacterLimitUiBehavior, chars: number };
 export function changeLimitConfig(payload: EditorLimit) {
+    const config = payload.ui ? payload : {...payload, ui: DEFAULT_UI_FOR_EDITOR_LIMIT};
```

Comment:
when writing ifs/ternaries against non-boolean values it'd be good to do `x != null` check instead of simply `if(x)`so it's more clear. For example an empty string is not null, but would evaluate to false in if/ternary.

Reply:
agree, but this should still evaluate to `false` if the string is empty. Although, that shouldn't happen unless some data gets corrupted

Reply:
string example wasn't supposed to apply here directly since `payload.ui` is an object. I just wanted to illustrate that `if (x)` is not the same as `if (x != null)`

Reply:
it is a string (`type CharacterLimitUiBehavior = 'highlight' | 'limit'`), that's why I did took it as an example